internal/generator/fluentd/output/security/security.go

```diff
@@ -10,6 +10,8 @@ import (
 
 type TLS bool
 
+type HostnameVerify bool
```

Comment:
I'm not sure I see the need for a HostnameVerify type. Just like the TLS type above.

Reply:
Just like other types in this package, this is a common type which captures the requirements. then each output package adds the template needed for generating conf.